src/adapter/breakpoints.ts

```diff
@@ -629,12 +638,6 @@ export class BreakpointManager {
       return;
     }
 
-    // Don't apply a custom breakpoint here if the user already has one.
-    const byPath = this._byPath.get(source.path) ?? [];
-    if (byPath.some(isSetAtEntry)) {
-      return;
-    }
```

Comment:
Why did we need to remove this?

Reply:
Sorry, this is not needed any more now that I'm using a unique id for the entry stop breakpoint. Restoring